lighthouse-core/report/html/renderer/util.js

```diff
@@ -66,6 +66,10 @@ class Util {
     if (!clone.configSettings.locale) {
       clone.configSettings.locale = 'en';
     }
+    if (!clone.configSettings.formFactor) {
+      // @ts-expect-error fallback handling for emulatedFormFactor
```

Comment:
can  you add a version comment like on the other comments? Someday we won't remember when this change happened :)